bin/sdl.cc

```diff
@@ -283,325 +281,218 @@ int main(int argc, char** argv)
-                std::vector<std::vector<unsigned int>>    out_hitId;
-                std::vector<std::vector<unsigned int>>    out_hitIdxs;
-                std::vector<std::vector<unsigned int>>    out_hitIndices_vec0;
-                std::vector<std::vector<unsigned int>>    out_hitIndices_vec1;
-                std::vector<std::vector<unsigned int>>    out_hitIndices_vec2;
-                std::vector<std::vector<unsigned int>>    out_hitIndices_vec3;
-                std::vector<std::vector<float>>    out_deltaPhi_vec;
-                std::vector<std::vector<float>>    out_ptIn_vec;
-                std::vector<std::vector<float>>    out_ptErr_vec;
-                std::vector<std::vector<float>>    out_px_vec;
-                std::vector<std::vector<float>>    out_py_vec;
-                std::vector<std::vector<float>>    out_pz_vec;
-                std::vector<std::vector<float>>    out_eta_vec;
-                std::vector<std::vector<float>>    out_etaErr_vec;
-                std::vector<std::vector<float>>    out_phi_vec;
-                std::vector<std::vector<int>>    out_charge_vec;
-                std::vector<std::vector<int>>      out_superbin_vec;
-                std::vector<std::vector<int8_t>>      out_pixelType_vec;
-                std::vector<std::vector<short>>    out_isQuad_vec;
-                std::vector<int>    evt_num;
-                std::vector<TString> file_name;
-                //std::vector<SDL::Event> events;
+
+    std::vector<std::vector<float>> out_trkX;
+    std::vector<std::vector<float>> out_trkY;
+    std::vector<std::vector<float>> out_trkZ;
+    std::vector<std::vector<unsigned int>> out_hitId;
+    std::vector<std::vector<unsigned int>> out_hitIdxs;
+    std::vector<std::vector<unsigned int>> out_hitIndices_vec0;
+    std::vector<std::vector<unsigned int>> out_hitIndices_vec1;
+    std::vector<std::vector<unsigned int>> out_hitIndices_vec2;
+    std::vector<std::vector<unsigned int>> out_hitIndices_vec3;
+    std::vector<std::vector<float>> out_deltaPhi_vec;
+    std::vector<std::vector<float>> out_ptIn_vec;
+    std::vector<std::vector<float>> out_ptErr_vec;
+    std::vector<std::vector<float>> out_px_vec;
+    std::vector<std::vector<float>> out_py_vec;
+    std::vector<std::vector<float>> out_pz_vec;
+    std::vector<std::vector<float>> out_eta_vec;
+    std::vector<std::vector<float>> out_etaErr_vec;
+    std::vector<std::vector<float>> out_phi_vec;
+    std::vector<std::vector<int>> out_charge_vec;
+    std::vector<std::vector<int>> out_superbin_vec;
+    std::vector<std::vector<int8_t>> out_pixelType_vec;
+    std::vector<std::vector<short>> out_isQuad_vec;
+    std::vector<int> evt_num;
+    std::vector<TString> file_name;
+
     // Looping input file
     while (ana.looper.nextEvent())
     {
 
         // if (ana.looper.getCurrentEventIndex() ==49) {continue;}
         std::cout << "PreLoading event number = " << ana.looper.getCurrentEventIndex() << std::endl;
```

Comment:
I would put this as well under `verbosity >= 1`. Or maybe even drop it all together, since this step happens really fast and, in >99% of the cases, without any issues.

Reply:
i am not quite sure i follow which line needs to be put under verbosity >=1.
Do you mean evt_num and file_name ? this is necessary for writing out ntuple.

Reply:
I think the confusion comes from the GitHub interface. When it shows a code block like this, the comment only refers to the last line shown - the others are for reference. In this comment, I was only referring to the last line:
```
std::cout << "PreLoading event number = " << ana.looper.getCurrentEventIndex() << std::endl;
```
This line "spams" us for no reason when we run the code.